pyodide_build/out_of_tree/venv.py

```diff
@@ -9,6 +9,29 @@
 from pyodide_build.common import exit_with_stdio
 from pyodide_build.logger import logger
 
+# A subset of supported virtualenv options that make sense in Pyodide's context.
+# Our aim will not be to support all of them, and some of them will never be in
+# the list, for example, --no-pip and so on. We provide these on a best-effort
+# basis as they should work and are easy to test.
+SUPPORTED_VIRTUALENV_OPTIONS = [
+    "--clear",
+    "--no-clear",
+    "--no-vcs-ignore",
+    "--symlinks",
+    # "--copies", "--always-copy", FIXME: node fails to invoke Pyodide
+    # "--symlink-app-data", FIXME: node fails to invoke Pyodide
```

Comment:
Noting for completeness: for supporting these two arguments, we'll definitely need to patch the Pyodide Python (which I need to learn more about).